.env.example

```
@@ -4,3 +4,4 @@ OPENAI_MODEL=gpt-4o-2024-08-06
 TELEGRAM_BOT_TOKEN=
 TELEGRAM_CHAT_ID=
 CALENDAR_ICS_URL=
+CALENDAR_REFETCH_INTERVAL=15
```

let's include the dimension in the variable name to avoid confusion

```suggestion
CALENDAR_REFETCH_INTERVAL_MIN=15

```